### PR TITLE
Add English functions introduction

### PR DESCRIPTION
**Pull request by superxi911**

### CODE REVIEW

**Comment by superxi911** [comment]
@gaocegege @zoumo @xiaomuqiao PTAL carefully

File: docs/functions.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Security scanning
+
+<div align="center">
+	<img src="./security.png" alt="security" width="500">
+</div>
+
+- Notifying via email 
+
+<div align="center">
+	<img src="./pagging.png" alt="pagging" width="500">
```
Comment (gaocegege) [inline docs/functions.md]: why the log in email doesn't have color now 🤔

File: docs/functions.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- Listing all services
+
+<div align="center">
+	<img src="./list_services.png" alt="list_services" width="500">
+</div>
+
+## Continus integration and security scanning
+
+All of the processes in workflow are visible. "prebuild" compile the executable files. "build" build the published image. "integration" execute the integrated test. "publish" publish the image and scan the vulnerabilities. "post build" do some relating operations after image published. "deploy" use the published image deploy a application to Kubernetes or any other container cloud platform. Cyclone can send email to notify the result of the workflow. All of the processes are shipped by container. It will wipe off the differences cause by environment.
```
Comment (gaocegege) [inline docs/functions.md]: I think "prebuild", "build" and so on should take the third person singular form of the verb. i.e. "prebuild" compile**s**

File: docs/functions.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	<img src="./logs.png" alt="logs" width="500">
+</div>
+
+- Security scanning
+
+<div align="center">
+	<img src="./security.png" alt="security" width="500">
+</div>
+
+- Notifying via email
```
Comment (gaocegege) [inline docs/functions.md]: I think Notifying is not suitable, maybe "sending the build log via email" is better?

File: docs/functions.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+<div align="center">
+	<img src="./list_services.png" alt="list_services" width="500">
+</div>
+
+## Continus integration and security scanning
+
+All of the processes in workflow are visible. "prebuild" compile the executable files. "build" build the published image. "integration" execute the integrated test. "publish" publish the image and scan the vulnerabilities. "post build" do some relating operations after image published. "deploy" use the published image deploy a application to Kubernetes or any other container cloud platform. Cyclone can send email to notify the result of the workflow. All of the processes are shipped by container. It will wipe off the differences cause by environment. 
+
+- Logs of the workflow
```
Comment (gaocegege) [inline docs/functions.md]: s/Logs/Log

File: docs/functions.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Function Introduction
+
+## Relating with VCS tools
+
+Cyclone can relate with various common VCS tools, such as git, svn, etc. After OAuth, it can pull codes from repository and create webhook. Whenever the user commit, submit a pull request or release a version to the repository, the webhook will trigger the CI/CD workflow.
```
Comment (gaocegege) [inline docs/functions.md]: I think it should be "Cyclone has been integrated with a variety of VCS tools", relate is not suitable.

**Comment by gaocegege** [comment]
And by the way, the images in readme is too small, Could you zoom in the webpage and screenshot again?

File: docs/functions.md (hunk 5 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md (hunk 2 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md (hunk 4 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md (hunk 3 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md (hunk 1 above)
Comment (superxi911) [inline docs/functions.md]: @xiaomuqiao has deleted the color from the log. Because it will interfere the display on web.

**Comment by superxi911** [comment]
@gaocegege The originals are clearer. They may be processed by GitHub. 😓

File: docs/functions.md (hunk 1 above)
Comment (gaocegege) [inline docs/functions.md]: Alright.

**Comment by gaocegege** [comment]
= = OK, it doesn't matter.

File: docs/functions.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+	<img src="./create_service.png" alt="create_service" width="500">
+</div>
+
+- Listing all services
+
+<div align="center">
+	<img src="./list_services.png" alt="list_services" width="500">
+</div>
+
+## Continus integration and security scanning
```
Comment (xiaomuqiao) [inline docs/functions.md]: s/scanning/detecting ?

File: docs/functions.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+All of the processes in workflow are visible. "prebuild" compiles the executable files. "build" builds the published image. "integration" executes the integrated test. "publish" publishes the image and scans the vulnerabilities. "post build" does some relating operations after image published. "deploy" uses the published image deploy a application to Kubernetes or any other container cloud platform. Cyclone can send email to notify the result of the workflow. All of the processes are shipped by container. It will wipe off the differences cause by environment.
```
Comment (xiaomuqiao) [inline docs/functions.md]: ' "build" builds the published image' ? published?

Comment (superxi911) [inline docs/functions.md]: s/compiles/compiles into

Comment (xiaomuqiao) [inline docs/functions.md]: s/'uses the published image deploy a application to'/'uses the published image to deploy or update a application into'

File: docs/functions.md (hunk 6 above)
Comment (superxi911) [inline docs/functions.md]: "scanning" is more suitable

File: docs/functions.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+# Function Introduction
+
+## Relating with VCS tools
+
+Cyclone has been integrated with a variety of VCS tools, such as git, svn, etc. After OAuth, it can pull codes from repository and create webhook. Whenever the user commit, submit a pull request or release a version to the repository, the webhook will trigger the CI/CD workflow.
```
Comment (Wumengna0402) [inline docs/functions.md]: 第二行“whenever the user commit”，commit加s＝》commits

File: docs/functions.md (hunk 7 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+- Resource quota of single building
+
+<div align="center">
+	<img src="./create_version.png" alt="create_version" width="500">
+</div>
+
+## Union publishing and dependency management
+
+Cyclone can manage multi-component united builds. It use the graphical user interface to display and manage the dependency of the components.
```
Comment (Wumengna0402) [inline docs/functions.md]: s/It use the graphical/It uses the graphical

File: docs/functions.md (hunk 7 above)
Comment (Wumengna0402) [inline docs/functions.md]: would/It can send email to/It would send email to

Comment (Wumengna0402) [inline docs/functions.md]: caused/cause/the differences caused by environment.

File: docs/functions.md (hunk 8 above)
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+- Sending the build log via email 
+
+<div align="center">
+	<img src="./pagging.png" alt="pagging" width="500">
+</div>
+
+## Resource management
+
+Cyclone separates the logic of scheduling and the building workflow. It also supports to add user worker nodes and various quota plans.
```
Comment (superxi911) [inline docs/functions.md]: 👌

File: docs/functions.md (hunk 7 above)
Comment (superxi911) [inline docs/functions.md]: 👌

Comment (superxi911) [inline docs/functions.md]: 👌

**Pull request merged**